src/e2e-test/features/gcs/source/GCSSourceToBigQuery.feature

```diff
@@ -285,3 +285,304 @@ Feature: GCS source - Verification of GCS to BQ successful data transfer
     Then Verify the pipeline status is "Succeeded"
     Then Get count of no of records transferred to target BigQuery Table
     Then Validate the values of records transferred from GCS bucket file is equal to the values of target BigQuery table
+
+    # New test scenarios added by Bijay on Nov 13
+  @GCS_TSV_TEST @BQ_SINK_TEST
+  Scenario: To verify Successful GCS to BigQuery file format selection using tsv
+    Given Open Datafusion Project to configure pipeline
+    When Select plugin: "GCS" from the plugins list as: "Source"
+    When Sink is BigQuery
```

Comment:
Use the new step similar to above for selecting the Bigquery plugin

Reply:
Added.